charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.3/src/templates/dotnet-imagestream.yaml

```diff
@@ -25,18 +25,18 @@ spec:
       sampleRef: dotnet-{{ $latestversion }}
     from:
       kind: ImageStreamTag
-      name: {{ $latestversion }}-{{ include "platformTag" $latest.platform }}
+      name: '{{ $latestversion }}'
     referencePolicy:
       type: Local
   {{- range $version, $info := $data.releases }}
   - name: {{ $version }}-{{ include "platformTag" $info.platform }}
     annotations:
-      openshift.io/display-name: .NET {{ $version }} ({{ $info.platform }})
+      openshift.io/display-name: .NET SDK {{ $version }} ({{ $info.platform }})
       openshift.io/provider-display-name: Red Hat, Inc.
       description: Build and run .NET {{ $version }} applications on {{ $info.platform }}. For more information
         about using this builder image, including OpenShift considerations, see {{ $info.sdkdoc }}.
       iconClass: icon-dotnet
-      tags: builder,dotnet
+      tags: builder,dotnet,hidden
```

Comment:
The hidden tag was moved to the ubi suffixed tags. This is expressing a preference for users to use `9.0` over `9.0-ubi8`.